### PR TITLE
Fix snapshots

### PR DESCRIPTION
**Pull request by niclim**
## Why
Describe the motivation for the changes.

Intercepts the test snapshot call to use sequential ID generation

There's snapshot changes in one of the tests `new request body has correct commands` - I'm not that familiar with the changes that's been happening - is this intended?

## What
What's changing? Anything of note to call out?

## Validation
* [ ] CI passes
* [ ] Verified in staging
* etc...

### CODE REVIEW

File: workspaces/ui-v2/src/lib/__tests/diff-helpers/fixture.ts [hunk 1]
```diff
@@ -32,6 +32,14 @@ export const testCase = (basePath: string) => async (
     require(path.join(__dirname + '/universes', basePath, name + '.json'))
   );
 
+  // Overwrite the learn_undocumented_bodies to ensure deterministic id generation
```
Comment (devdoshi) [inline workspaces/ui-v2/src/lib/__tests/diff-helpers/fixture.ts:35]: cool, i'm fine with this, but ideally we'd "just" pass the id generation strategy from the entry point i.e. when creating an inmemory opticcontext

Comment (devdoshi) [reply workspaces/ui-v2/src/lib/__tests/diff-helpers/fixture.ts:35]: the entry point should be trusted to set the right environment for clocks, ids, etc.

File: workspaces/ui-v2/src/lib/__tests/diff-helpers/__snapshots__/new-regions-interpretations.test.ts.snap [hunk 2]
```diff
@@ -7,116 +7,81 @@ Object {
...
       },
     },
     Object {
       "AddShape": Object {
         "baseShapeId": "$object",
         "name": "",
-        "shapeId": "shape_6708",
-      },
-    },
-    Object {
```
Comment (niclim) [inline workspaces/ui-v2/src/lib/__tests/diff-helpers/__snapshots__/new-regions-interpretations.test.ts.snap:41]: @devdoshi here's the shape diff from the snapshot - is this diff related to the diff engine changes that's been going on?

**Review by devdoshi — APPROVED** [review]

File: workspaces/ui-v2/src/lib/__tests/diff-helpers/__snapshots__/new-regions-interpretations.test.ts.snap (hunk 2 above)
Comment (devdoshi) [reply workspaces/ui-v2/src/lib/__tests/diff-helpers/__snapshots__/new-regions-interpretations.test.ts.snap:41]: yeah I imagine it'll be in flux as we fix some bugs around these areas, we'll be responsible for verifying the final snapshots

**Comment by JaapRood** [comment]
Rebased #788 on top of this, still fails the snapshots :(

**Pull request merged**